<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel Manager Admin 2011 [bienvenue au service administration]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/logo_a.png</normaloff>:/images/logo_a.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionSe_connecter"/>
    <addaction name="actionSe_d_connecter"/>
    <addaction name="separator"/>
    <addaction name="actionActiver_son_compte"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEditer">
    <property name="title">
     <string>Editer</string>
    </property>
    <addaction name="actionChanger_password"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAide_en_ligne"/>
    <addaction name="separator"/>
    <addaction name="actionA_propos_de_Hotel_Manager_Plus"/>
   </widget>
   <widget class="QMenu" name="menuMessagerie">
    <property name="title">
     <string>Messagerie</string>
    </property>
    <addaction name="actionEcrire_et_recevoir_des_messages"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEditer"/>
   <addaction name="menuMessagerie"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSe_connecter"/>
   <addaction name="actionSe_d_connecter"/>
   <addaction name="separator"/>
   <addaction name="actionEcrire_et_recevoir_des_messages"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionChanger_password">
   <property name="text">
    <string>Editer votre compte...</string>
   </property>
   <property name="toolTip">
    <string>Changer le mot de passe du compte administrateur en cours</string>
   </property>
  </action>
  <action name="actionSe_connecter_au_service_R_ception">
   <property name="text">
    <string>Se connecter au service Réception</string>
   </property>
  </action>
  <action name="actionSe_connecter">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/connecter.png</normaloff>:/images/connecter.png</iconset>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </action>
  <action name="actionSe_d_connecter">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/déconnecter.png</normaloff>:/images/déconnecter.png</iconset>
   </property>
   <property name="text">
    <string>Se déconnecter</string>
   </property>
  </action>
  <action name="actionActiver_son_compte">
   <property name="text">
    <string>Activer un nouveau compte...</string>
   </property>
  </action>
  <action name="actionEcrire_et_recevoir_des_messages">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/msgs2.png</normaloff>:/images/msgs2.png</iconset>
   </property>
   <property name="text">
    <string>Envoyer et/ou recevoir des messages</string>
   </property>
  </action>
  <action name="actionAide_en_ligne">
   <property name="text">
    <string>Aide en ligne</string>
   </property>
  </action>
  <action name="actionA_propos_de_Hotel_Manager_Plus">
   <property name="text">
    <string>A propos de Hotel Manager Plus...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
